### ISSUE TITLE
Server randomly crashes when using functions

### ISSUE THREAD

**Issue by mikk3612** [SkriptLang/Skript#1594]
Before going any further, it is important to note that this does not occur whenever I use a function, yet only once in a while, when no function should be called. I'd say every 10-15 minutes. The functions shouldn't be called, since they're only GUI related, and it crashes even though I do not open the GUI (somehow?).

This issue seems to be related with a threading issue, yet not the same errors:
https://github.com/SkriptLang/Skript/issues/1591

Version:
alpha5

Error:
https://hastebin.com/iluyaholek.md

Thank you.

**Comment by ham1255** [comment]
get the skript code

**Comment by mikk3612** [comment]
`
function ClearProfessionMenu(p: player, n: number):
	FormatMenu({_p},"Clear Profession Slot %{_n}%")
	wait 2 ticks
	format gui slot 49 of {_p} with {gui::mainmenu} to run function OpenProfessionBook({_p})

	format gui slot 21 of {_p} with {gui::prof::profconfirm} to run function SetProfession({_p},{_n},"null",1)
	format gui slot 23 of {_p} with {gui::prof::profcancel} to run function OpenProfessionBook({_p})`

It's all very simple methods that somewhat manipulate a GUI (Using Tuske), but Tuske doesnt seem to be the thing crashing the server, though it's not 1.13 compliant.

**Comment by bensku** [comment]
Actually this is still duplicate of #1591.
